pkg/platform/packet/packet.go

```diff
@@ -667,3 +691,20 @@ func checkResFormat(reservationIDs map[string]string, name, errorPrefix, resPref
 
 	return diagnostics
 }
+
+func areUniqueStrings(strs []string) ([]string, error) {
```

Comment:
I would expect `areUniqueStrings` to return a bool, not some list of strings.

As I said before, returning list of distinct elements and comparing it higher-level should make thing composed better.

Reply:
With bool, we can't specifically tell user which CIDR is repeating. There will be just a cryptic message that something is repeated.

Reply:
Yes, so as I said before, we should make this function return distinct element without error and make error checking in a different place.

Reply:
Then what is the point of this function? If there is going to be another iteration to find out which element is repeating?

Reply:
Yes, it will be used once for validation and once for side-effect. + function allows you to isolate and name block of the code, so even if it would be used once, it's still useful to have a generic function like this extracted.

Reply:
I also think what @invidian suggests is more clear:

```
func areUniqueStrings(strs []string) bool;
func uniqueStrings(strs []string) []string;
```

Reply:
Created a `func` which returns repeated value, figured we only need that.